float32 hs_pos_x
float32 hs_pos_y
float32 hs_pos_z
float32 hs_rot_x
float32 hs_rot_y
float32 hs_rot_z
float32 hs_rot_w
float32 lcon_pos_x
float32 lcon_pos_y
float32 lcon_pos_z
float32 lcon_rot_x
float32 lcon_rot_y
float32 lcon_rot_z
float32 lcon_rot_w
float32 rcon_pos_x
float32 rcon_pos_y
float32 rcon_pos_z
float32 rcon_rot_x
float32 rcon_rot_y
float32 rcon_rot_z
float32 rcon_rot_w